src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationMiddleware.cs

```diff
@@ -46,9 +45,14 @@ public class OpenIdConnectAuthenticationMiddleware : AuthenticationMiddleware<Op
         {
             _logger = loggerFactory.CreateLogger<OpenIdConnectAuthenticationMiddleware>();
 
+            if (string.IsNullOrEmpty(Options.SignInScheme))
+            {
+                Options.SignInScheme = externalOptions.Options.SignInScheme;
+            }
```

Comment:
If we decide to remove `SignInScheme` from the OIDC options to favor the global `ExternalAuthenticationOptions`, we must be aware that it will be far less granular: if we have 2 OIDC middleware in a pipeline, we won't be able to define 2 different cookie middleware for these instances.

We could also make `OpenIdConnectAuthenticationOptions` inherit from `ExternalAuthenticationOptions`.